plenum/server/replica.py

```diff
@@ -2333,6 +2346,7 @@ def _gc_before_new_view(self):
         # Clear any checkpoints, since they are valid only in a view
         self._gc(self.last_ordered_3pc)
         self.checkpoints.clear()
+        self.consensus_provider.remove_all_checkpoints()
```

Comment:
PBFT view change doesn't assume cleaning of old checkpoints

Reply:
comment added